luma/core/interface/serial.py

```diff
@@ -428,7 +426,7 @@ def output(self, pin, value):
 
         self._gpio.write(self._data)
 
-    def cleanup(self):
+    def cleanup(self, pin):
```

Comment:
changing the public api should be avoided.

Reply:
This is in following class:
```
class __FTDI_WRAPPER_GPIO:
    """
    Adapter for FTDI to RPI.GPIO. Not for direct public consumption
    """
```

Reply:
ah yes, sorry for the noise.